at-luantran/Week3/2018_07_05_exercise3_rubyday3.rb

```diff
@@ -0,0 +1,43 @@
+class HtmlToText
+  def get_html(html)
+    arr_char = html.split('')
+    @string = ''
+    stack = []
+    i = 0
+    check = false
+    while i < arr_char.length do
+      if(arr_char[i..i+4].join == '<div>')
+        stack.push('div')
+        if(check == true)
+          @string += "\n"
+          check = false
+        end
+        i += 5 
+      elsif(arr_char[i..i+3].join == '<br>')
+        if(check == true)
+          @string += "\n"
+          check = false
+        end
+        i += 3
+      elsif(arr_char[i..i+5].join == '</div>')
+        if(stack.pop == 'div' && check == true)
+          @string += "\n"
+          check = false
+        end
+        i += 6
+      else
+        @string += arr_char[i]
+        check = true
+        i += 1
+      end
+    end
+  end
+
+  def show_view
+    puts @string
+  end
+end
+
+obj = HtmlToText.new()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/838e41b2d4bb928094b6df310b810687d65450b5/at-luantran/Week3/2018_07_05_exercise3_rubyday3.rb#L41"></span>Style/MethodCallWithoutArgsParentheses: Do not use parentheses for method calls with no arguments. (https://github.com/rubocop-hq/ruby-style-guide#method-invocation-parens)</td>
    </tr>
  </tbody>
</table>